tests/unit/protobuf/test_proto_file_element.py

```diff
@@ -18,7 +18,7 @@
 
 import copy
 
-location: Location = Location.get("some/folder", "file.proto")
+location: Location = Location("some/folder", "file.proto")
```

Comment:
nitpick (non-blocking): Probably unnecessary type hint? (same in some other modules in same package).